### DISCUSSION TITLE
IOS app offline support

### DISCUSSION THREAD

**Discussion by n7ihq** [AppFlowy-IO/AppFlowy#2437] [General]
Will the IOS app function offline? This is very useful for viewing and entering notes and tasks when cell phone coverage is marginal or unavailable. TickTick and Todoist task management apps have offline support.

**Comment by annieappflowy** [comment]
By default, AppFlowy Mobile is offline unless you choose to sync across devices or subscribe to our upcoming cloud solutions

**Reply by lyziam** [reply]
Appflowy is excellent! I'm looking forward to AppFlowy Mobile ! Will it be released in May?

**Reply by annieappflowy** [reply]
Thanks for your support! [appflowy_editor](https://pub.dev/packages/appflowy_editor) will support mobile by the end of Q2, and AppFlowy will release a mobile app after, very likely in Q3

**Reply by zparihar** [reply]
Hi @annieappflowy ,

Forgive my ignorance, but how will it sync?  Do you have a Self-hosted Sync Server (kind of like Joplin-server)? 
Or are you using basic file-system sync?

Thanks!
